addons/web/static/src/webclient/actions/action_service.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
encoding declaration is useless, since this template will always be parsed by Owl, which has utf-8 as default encoding
```suggestion
<?xml version="1.0">
```